src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -1259,20 +1259,23 @@ public void Cast()
                 // (F) [A] () => { }
                 Diagnostic(ErrorCode.ERR_UnexpectedToken, "(F) [A] ()").WithArguments("=>").WithLocation(1, 1));
```

Comment:
this restores the parsing behavior (in an error recovery case) here to the prior behavior we had here that i changed in the original parsing PR: https://github.com/dotnet/roslyn/pull/66417/files#diff-e77ec53b27dcea093de7ebc648dd17ff814931a0e44e001d5b6ab5d0a3403f0bR1255